src/components/message/message.scss

```diff
@@ -0,0 +1,60 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Display message and close button in a centered row.
+ * 2. Push message and close button as far apart as possible.
+ * 3. Condense line height in case it breaks to multiple lines.
+ *
+ * @TODO
+ * 1. Style `:focus` state
+ */
+
+.c-message {
+  align-items: center; /* 1 */
+  background: colors.$gray-light;
+  border-top: 4px solid colors.$gray-dark;
+  color: colors.$text-dark;
+  display: flex; /* 1, 2 */
+  justify-content: space-between; /* 2 */
+  line-height: ms.step(1, 1); /* 3 */
+  padding: ms.step(-8) ms.step(0);
+}
+
+.c-message--warning {
+  background: colors.$fuchsia;
+  border-top: 4px solid colors.$fuchsia-darker;
+  color: colors.$text-light-emphasis;
+}
+
+/**
+ * Inherit parent text color for links when blue is hideous against background.
+ */
+
+.c-message--warning a {
+  color: inherit;
+}
```

Comment:
Would this be better accomplished via a theme?

Reply:
🤔 Perhaps?

Since this pattern is likely to sit above all all page content, instead of within page content, I wasn't sure theme variants were appropriate here.